packages/ckeditor5-image/tests/imageresize/imageresizeediting.js

```diff
@@ -136,6 +136,67 @@ describe( 'ImageResizeEditing', () => {
 		} );
 	} );
 
+	describe( 'conversion (inline images)', () => {
+		beforeEach( async () => {
+			editor = await createEditor();
+		} );
+
+		it( 'upcasts 100px width correctly', () => {
+			editor.setData(
+				`<p>Lorem <span class="image-inline" style="width:100px;"><img src="${ IMAGE_SRC_FIXTURE }"></span> ipsum</p>`
+			);
+
+			expect( editor.model.document.getRoot().getChild( 0 ).getChild( 1 ).getAttribute( 'width' ) ).to.equal( '100px' );
+		} );
+
+		it( 'upcasts 50% width correctly', () => {
+			editor.setData( `<p>Lorem <span class="image-inline" style="width:50%;"><img src="${ IMAGE_SRC_FIXTURE }"></span> ipsum</p>` );
+
+			expect( editor.model.document.getRoot().getChild( 0 ).getChild( 1 ).getAttribute( 'width' ) ).to.equal( '50%' );
+		} );
```

Comment:
What about upcasting just `<img style="width: ...">`? Inline images don't exist in the editor data pipeline as `<span><img /></span>` – this is an editing pipeline representation.

Reply:
Even the next text confirms this :point\_down: